packages/xrpl/src/models/utils/flags.ts

```diff
@@ -36,8 +36,12 @@ export function parseAccountRootFlags(
 ): AccountRootFlagsInterface {
   const flagsInterface: AccountRootFlagsInterface = {}
 
-  Object.keys(AccountRootFlags).forEach((flag) => {
-    if (isFlagEnabled(flags, AccountRootFlags[flag])) {
+  // If we use keys all will be strings and enums are reversed during transpilation
+  Object.values(AccountRootFlags).forEach((flag) => {
```

Comment:
What does this change do?

Reply:
This was to fix an issue with flags.  enums get turned into an object with double the values.  Key value like you define and then the reverse of value key.  This meant that isFlagEnabled was being passed strings for the flag value which was blowing up isFlagEnabled after making them be `bigint`